src/proto.rs

```diff
@@ -970,9 +961,17 @@ pub struct CollectedClientData {
     pub challenge: Base64UrlSafeData,
     /// The rp origin as the browser understood it.
     pub origin: String,
+    /// The inverse of the sameOriginWithAncestors argument value that was
+    /// passed into the internal method.
+    #[serde(rename = "crossOrigin", skip_serializing_if = "Option::is_none")]
+    pub cross_origin: Option<bool>,
```

Comment:
Do we need to perform any validation around the option and it's usage? We should understand what it does ....

Reply:
I think it indicates if the javascript context that created the public key credential had a different origin than the origin that the credential is created for.

We could consider adding a predicate `allow_cross_origin` in the `WebauthnConfig` trait (defaulting to `false` always) that the user can set to indicate whether they want to accept such credentials.